fvm/evm/offchain/utils/collection_test.go

```diff
@@ -0,0 +1,98 @@
+package utils_test
```

Comment:
Running into some error when running this tests. Did we forget to commit somecode?
```
fvm/evm/offchain/utils/collection_test.go:1: : # github.com/onflow/flow-go/fvm/evm/offchain/utils_test [github.com/onflow/flow-go/fvm/evm/offchain/utils.test]
fvm/evm/offchain/utils/collection_test.go:19:2: "github.com/onflow/flow-go/fvm/evm/testutils" imported and not used
fvm/evm/offchain/utils/collection_test.go:70:10: undefined: NewTestStorageProvider
fvm/evm/offchain/utils/collection_test.go:72:19: cr.ReplayBlock undefined (type *"github.com/onflow/flow-go/fvm/evm/offchain/sync".Replayer has no field or method ReplayBlock)
fvm/evm/offchain/utils/collection_test.go:80:55: too many arguments in call to bp.OnBlockExecuted
        have (uint64, unknown type)
        want (*"github.com/onflow/flow-go/fvm/evm/events".BlockEventPayload) (typecheck)
```

Reply:
let me check